src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService service = new UserService();
+        User userService = service.findByEmail(email);
```

Comment:
it's better to change naming for variables
```suggestion
        UserService userService = new UserService();
        User user = service.findByEmail(email);
```